pkg/kubelet/cm/types.go

```diff
@@ -37,6 +38,8 @@ type ResourceConfig struct {
 	// Memory node sets.
 	// List format: http://man7.org/linux/man-pages/man7/cpuset.7.html#FORMATS
 	CpusetMems *string
+	// Hugetlb limit (in bytes)
+	HugetlbLimit []*hugepage.HugepageLimit
```

Comment:
I'd prefer to leave `HugetlbLimit` since in `libcontainer/config/cgroup_unix.go` this is also without 's':
```
// Hugetlb limit (in bytes)
HugetlbLimit []*HugepageLimit `json:"hugetlb_limit"`
```
Hope you dont mind ?